components/TextFields/src/MDCTextField.m

```diff
@@ -34,6 +34,15 @@ @interface MDCTextField ()
 
 @property(nonatomic, strong) MDCTextInputCommonFundament *fundament;
 
+/** 
+ 
+ This constraint controls the default center Y of the underline view. 
+
+ Default is from the top of the text field with constant of height of input's font line-height +
+ half padding.
```

Comment:
Too complicated to understand in a single go. Maybe add ", e.g. below the text field at a distance of one-half of the padding"?